### PR TITLE
Plugin: Double Click Tab

### PR DESCRIPTION
**Pull request by Quorafind**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/Quorafind/Obsidian-Double-Click-Tab

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by Quorafind** [comment]
@joethei fixed

**Comment by liamcain** [comment]
- [applyDebounceTimer](https://github.com/Quorafind/Obsidian-Double-Click-Tab/blob/294cd6909ce7f1dcd54d4e9a7c3e55c32ee2d471/dblClickTabIndex.ts#L138) why not just use Obsidian's [debounce](https://github.com/obsidianmd/obsidian-api/blob/1b4f6e2e5753c8ac2d538bc5d55c38f228450cb2/obsidian.d.ts#L772) function?

File: community-plugins.json [hunk 1]
```diff
@@ -5424,5 +5424,13 @@
         "description": "A plugin allows you to auto add current time to new bullet line.",
         "repo": "quorafind/obisidna-new-bullet-with-time",
         "branch": "master"
+    },
+    {
+        "id": "obsidian-double-click-tab",
```
Comment (liamcain) [inline community-plugins.json:5429]:
```suggestion
        "id": "double-click-tab",
```

**Comment by liamcain** [comment]
Looks like there's an issue with the manifest pointing to the wrong release. Ping me here when it's fixed.

**Comment by Quorafind** [comment]
> Looks like there's an issue with the manifest pointing to the wrong release. Ping me here when it's fixed.

@liamcain fixed

**Pull request merged**